README.md

```diff
@@ -136,21 +137,6 @@ or to run a single test:
 vagrant ssh -c "cd /vagrant/build && make libuavcan_test && ./libuavcan/libuavcan_test --gtest_filter=Node.Basic"
 ```
 
-### Developing with Eclipse
-
-An Eclipse project can be generated like that:
-
-```bash
-cmake ../../libuavcan -G"Eclipse CDT4 - Unix Makefiles" \
-                      -DCMAKE_ECLIPSE_VERSION=4.3 \
-                      -DCMAKE_BUILD_TYPE=Debug \
-                      -DCMAKE_CXX_COMPILER_ARG1=-std=c++11
-```
-
-Path `../../libuavcan` in the command above points at the directory where the top-level `CMakeLists.txt` is located;
-you may need to adjust this per your environment.
-Note that the directory where Eclipse project is generated must not be a descendant of the source directory.
-
```

Comment:
why remove though

Reply:
Because it's outdated, I've never tested it, and I don't want to support it. I used Eclipse for years but I gave up on it about six months ago. That project is such a hot-mess and vscode is almost at parity for C++ now so if we are going to provide help for an editor it should be that one. Frankly I'd rather just remain IDE friendly but agnostic. No information is better then wrong information.

Reply:
![image](https://user-images.githubusercontent.com/3298404/49393959-62f2a380-f73b-11e8-8f9b-e7e4aa41390b.png)

Reply:
I've been using CLion lately, it's mostly harmless and its embedded indexer can actually parse most of C++ correctly. Not sure about vscode. The question of "why aren't there any decent C++ IDE in 2018" keeps me up at night though.